packages/cozy-stack-client/src/FileCollection.js

```diff
@@ -417,6 +417,18 @@ class FileCollection extends DocumentCollection {
     return this.stackClient.fetchJSON('POST', uri`/files/trash/${id}`)
   }
 
+  /**
+   * Copy a file.
+   *
+   * @param {string} id   - The file's id
```

Comment:
```
@param {FileAttributes['id']} id   - The file's id
```

I know this is not used anywhere else, but let's make the code better ;)

Reply:
I tried this way but got

`JSDOC_ERROR: ERROR: Unable to parse a tag's type expression for source file /Users/theo/Cozy/cozy-client/packages/cozy-stack-client/src/FileCollection.js in line 420 with tag title "param" and text "{FileAttributes['id']} id   - The file's id": Invalid type expression "FileAttributes['id']": Expected "!", "#", "$", "(", "-", ".", "/", "0", ":", "<", "=", "?", "@", "[]", "\\", "_", "|", "~", "‌", "‍", Unicode combining mark, Unicode decimal number, Unicode letter number, Unicode lowercase letter, Unicode modifier letter, Unicode other letter, Unicode punctuation connector, Unicode titlecase letter, Unicode uppercase letter, [1-9], or end of input but "[" found.`

Tried some other ways too without success for the moment